src/components/Tiles.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+const Tiles = ({title, image, text}) => {
+    return (
+        <>
+            <section>
+                <h2> {title} </h2>
+            </section>
+            <section>  <p> {text}</p></section>
+            <section><img src={image} alt=""/></section>
+
+        </>
+    );
```

Comment:
- Het was voor deze opdracht niet nodig om hier 3 verschillende section tags te gebruiken. Dit kan gewoon allemaal in 1 section tag. 
- Hierdoor worden <> </> ook overbodig. 
- Dit is ook 1 van de redenen waarom je indeling van de pagina door elkaar gaat.
- Het is de bedoeling dat deze component meerdere paragrafen kan weergeven. Dit werkt met jouw code niet. Je kan dit oplossen door de propertie {children} te gebruiken (zie uitwerkingen).
- De alt tag is leeg in het component.